svelte/src/libs/stores/tabs.ts

```diff
@@ -0,0 +1,12 @@
+import { writable } from "svelte/store";
+
+export default function initTabStore() {
+  const activeTab = writable<string>();
```

Comment:
we probably want to explicitly define all tab types rather than just string

Reply:
done